.gitignore

```diff
@@ -23,6 +23,10 @@ coverage
 # Compiled Dirs (http://nodejs.org/api/addons.html)
 build/
 dist/
+public/bundle.js
 
 # Jest
 globalConfig.json
+
+# JUnit
+artifacts/
```

Comment:
Is this for Jenkins...Deployment?

Reply:
Used for Gitlab CI JUnit reports. As a report gets created every time you run jest, I've ignored the path.